--- a/emoncmsHub
+++ b/emoncmsHub
@@ -23,10 +23,9 @@ PATH=/sbin:/usr/sbin:/bin:/usr/bin
 DESC="OpenEnergyMonitor Gateway"
 NAME=oemgateway
 # Enter your installation path here
-DAEMON=/enter/your/installation/path/here/$NAME
+DAEMON=/usr/share/emoncmsHub/$NAME
 # Specify command line arguments here
-DAEMON_ARGS="--logfile /var/log/oemgateway/oemgateway.log"
-#DAEMON_ARGS="--logfile /var/log/oemgateway/oemgateway.log --config-file /path/to/config/file"
+DAEMON_ARGS="--logfile /var/log/emoncmsHub/emoncmsHub.log --config-file /etc/emoncmsHub/emoncmsHub.conf"
 DAEMONUSER="pi"
 PIDFILE=/var/run/$NAME.pid
 SCRIPTNAME=/etc/init.d/$NAME
